src/js/components/Menu.js

```diff
@@ -123,6 +124,16 @@ var MenuDrop = React.createClass({
     return true;
   },
 
+  _renderNestedMenuLinks: function (menuItems) {
+    if (isArray(menuItems.props.children)) {
```

Comment:
I think the children should be accessed via: https://facebook.github.io/react/docs/top-level-api.html#react.children